ui/src/components/table/QTable.json

```
@@ -788,7 +788,17 @@
         "cols": {
           "type": "Object",
           "desc": "Column definitions",
-          "__exemption": [ "examples" ]
+          "__exemption": [ "examples" ],
+          "definition": {
+            "name": {
+              "type": "String",
+              "desc": "Column name"
+            },
```

Let's match the 'desc' with 'columns'.
```suggestion
            "name": {
              "type": "String",
              "desc": "Unique id, identifies the column",
              "examples": [ "desc", "userId" ]
            },
```
Also, `String` wouldn't work without `examples`, it's a required property in the schema.